services/common/src/components/forms/Step.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from "react";
```

Comment:
One of many, many components to get moved or copied to common

Reply:
More more more!